### DISCUSSION TITLE
Version 10 with maplibre working?

### DISCUSSION THREAD

**Discussion by thertzelle** [rnmapbox/maps#1982] [Q&A]
I followed all the directions for setting up my react native project with the last main branch, added the code to get a basic map working (as provided in the readme), but when the app launches, it just crashes right away without any errors in any console.

Has anyone else gotten it to work successfully? Maybe a reference repo of a basic react native app with a basic map working?

**Comment by mfazekas** [comment]
@thertzelle iOS/android?!
